staging/src/k8s.io/apiserver/pkg/endpoints/handlers/fieldmanager/fieldmanager_test.go

```diff
@@ -173,3 +175,86 @@ func TestApplyDoesNotStripLabels(t *testing.T) {
 		t.Fatalf("labels shouldn't get stripped on apply: %v", m)
 	}
 }
+
+func BenchmarkApplyNewObject(b *testing.B) {
+	f := NewTestFieldManager()
+
+	obj := &corev1.Pod{}
+
+	b.ReportAllocs()
```

Comment:
Probably doesn't matter on these timescales, but technically a b.Reset() might be good to exclude the time it takes to construct `f`. (below, too)

Reply:
It doesn't hurt to have it ..